package.json

```diff
@@ -36,7 +36,7 @@
     "check": "flow check",
     "check:ts": "dtslint src",
     "check:cover": "node resources/check-cover.js && nyc report --nycrc-path .nycflowrc.yml",
-    "check:spelling": "cspell \"./{src/**/,resources/**/}*.{js,ts,md,graphql}\"",
+    "check:spelling": "cspell \"./{src/**/,resources/**/}*.{js,d.ts,md,graphql}\"",
```

Comment:
Why you disabled `cspell` on tests?

Reply:
Because it was overdoing it :D I got spelling errors on GraphQL schema definitions, like `geocode`

Reply:
So why it was passing on Flow tests and what changed?

Reply:
No idea actually. 
I turned it back on, and got this errors:
```
/Users/dotansimha/Dev/graphql-js/src/execution/__tests__/variables-test.ts:134:28 - Unknown word (structs)
/Users/dotansimha/Dev/graphql-js/src/utilities/__tests__/buildASTSchema-test.ts:1087:36 - Unknown word (behaviour)
/Users/dotansimha/Dev/graphql-js/src/utilities/__tests__/buildClientSchema-test.ts:431:18 - Unknown word (geocode)
/Users/dotansimha/Dev/graphql-js/src/utilities/__tests__/buildClientSchema-test.ts:432:9 - Unknown word (geocode)
/Users/dotansimha/Dev/graphql-js/src/utilities/__tests__/printSchema-test.ts:632:43 - Unknown word (behaviour)
/Users/dotansimha/Dev/graphql-js/src/utilities/__tests__/printSchema-test.ts:634:39 - Unknown word (behaviour)
/Users/dotansimha/Dev/graphql-js/src/utilities/__tests__/printSchema-test.ts:853:42 - Unknown word (behaviour)
/Users/dotansimha/Dev/graphql-js/src/utilities/__tests__/printSchema-test.ts:855:38 - Unknown word (behaviour)
```

With `master` it seems fine. maybe it skips those files somehow? 

I fixed some of them and added 2 to the `cspell` ignore list, is that ok?

Reply:
Please revert this change and just temporarily add them to `cspell.json` as a separate group with a comment referencing this thread.
I made 5 min investigation and it looks like not every dictionary that tied to `javascript` also tied to `typescript`, e.g.
https://github.com/streetsidesoftware/cspell-dicts/blob/20fc90e1e2f7408fcc797616972467cfa99038ec/packages/fullstack/cspell-ext.json#L23
So we don't need to change our descriptions just contribute a fix to `cspell-dicts` and remove this list from cspell.json after it would be merged.